sidekick_core/lib/sidekick_core.dart

```diff
@@ -284,8 +284,20 @@ class SidekickCommandRunner<T> extends CompletionCommandRunner<T> {
     final unmount = mount(debugName: args.join(' '));
 
     ArgResults? parsedArgs;
+    Future<void Function()?>? updateCheck;
```

Comment:
A `Future<void>` should be sufficient. The `void Function()` seems redundant

Reply:
I updated it to a String? :)